### PR TITLE
User roles updates

### PR DESCRIPTION
**Pull request by noahwc**
## Description of the change

> Description here

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation or Development tools (readme, specs, tests, code formatting)

## Links

- Jira issue number: [DEV-3342](https://saasquatch.atlassian.net/browse/DEV-3342)
- Process.st launch checklist: https://app.process.st/runs/User-Roles-Project-Workflow-rGxNj2YNcKI5vZppaUpKpQ

### Development

- [ ] [Prettier](https://www.npmjs.com/package/prettier) was run (if applicable)
- [ ] The behaviour changes in the pull request are covered by specs
- [ ] All tests related to the changed code pass in development

### Paperwork

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has a Jira number
- [ ] This pull request has a Process.st launch checklist

### Code review

- [ ] Changes have been reviewed by at least one other engineer
- [ ] Security impacts of this change have been considered

### CODE REVIEW

File: packages/visual-dev/CHANGELOG.md [hunk 1]
```diff
@@ -12,9 +12,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+  - \<Badge>
+    - Add "small" badge size
+  - \<IconButton>
+    - Fix cursor not entering pointer mode when hovering button
+  - \<Datatable>
+    - Support for custom empty state graphics
+  - \<DisplayDropdown>
+    - Add a new style of select/dropdown component, typically used in table rows
+  - \<GlobalStyle>
+    - Add tokens for surface dark and surface overlay colors
```
Comment (noahwc) [inline packages/visual-dev/CHANGELOG.md:28]: @tdgao please add a changelog entry for DisplayDropdown

File: packages/visual-dev/src/components/Graphics.tsx [hunk 2]
```diff
@@ -31,3 +31,35 @@ export const DataGraphic = (
...
...zPiruneHURUTkSPiv567OHvlz8qRN4eIfQOkGV5PRHFXfy6WIeIfiDL8nq97YhGRNO9CxRF+S9mfkpvOwTdQ0R/kGX5W3rbEa0IoXcBM5OiKG8AuF9vWwRdslaW5QdFd6tzhNC7gZlNqqp+yJ2EsxboCxF9ajabbyPSLx5bLCCEHgTMnKQoymYAM/W2RRDAAVmWFxNRo96GRDtiMC4IiKhRluUVAA7obYvgGgdkWV4hRB4cQuhBQkQ1/trjU71tiXeI6FP/b1Gjty2xghB6DyCiRrPZfBuAtXrbEses9ffJRU3eA4TQewgRuf0jvH/Q25Z4wz+F9qAYeOs5Qui9gIhYluVvSZL0fTGlE3n8wSO+L8vyt8T17h1i1D1EFEVZwcxrIHzjI0UNET0qPN5CQwg9DDBzlqIob0Ksegs3O2RZvl/4roeOaLqHASIqk2V5EYBfiKZl6Piv4S/8S02FyMOAqNHDjKIoy5j5NQDpetsSo1QT0SMiaER4EUKPAMw8QFXV5wF8hZnFNQ4Cfy3+stls/n8i/FP4ETdhBHG73XM0TfszM0/X25ZohogOSZL0DRGtNXIIoUcYZjaoqvp1AD9l5hS97YkmqDmN0Q/MZvNfRNinyCKE3kcw8xBVVX8M4DFmNuttj54QkQrgb2az+YdEVKW3PfGAEHofw8xDVVX9LoDVzBxXuZqJqAnAi2az+VdEVKG3PfGEELpOMHO6oihPE9E3mTlZb3siCRHZmfmPsiz/noiq9bYnHhFC1xlmTnG73U8w8xeZeYLe9oQTIiogoldNJtML/v64QCeE0KMIt9s9Q9O0R5n5AcTuPHw1Eb0pSdIak8l0UG9jBM0IoUchzGxUVXU5gEcBLGdmm942dQUROQBsALDGbDZvICKv3jYJAhFCj3KY2ej1emdpmraYmZcAmM/Msp42EZECYCcRbZIkabPRaNwvxB3dCKHHGMxsVhRlLoBFRJQPYCyA0cxsikR5/rXfZwEUMvMxAFtkWd7tnyITxAhC6P0Av1POSDSL/urfIAA2ZrYRUVKr1zb/MQ5/k9vBzI1XX6M5fXQRgEIAhWaz+aJwZhEIBAKBQCAQCAQCgUAgEAgEAoFAIBAIBAKBQCAQCAQCgUAgEAgEAoFAIBAIBAKBQCAQCAQCgUAgEAgEAoFAIBAIdOP/AGQBsyAFwcunAAAAAElFTkSuQmCC"
```
Comment (zachharrison) [inline packages/visual-dev/src/components/Graphics.tsx:61]: Are these graphics needed? I don't seem them being used anywhere, also we might be able to delete the enormous string of characters being passed to the `xlinkHref` attribute

**Review by zachharrison — COMMENTED** [review]
Left one comment, but besides that stories and code looks good

File: packages/visual-dev/src/components/Graphics.tsx (hunk 2 above)
Comment (noahwc) [reply packages/visual-dev/src/components/Graphics.tsx]: Confirmed not needed and removed

**Review by zachharrison — APPROVED** [review]
Looks good to me

**Pull request merged**